### PR TITLE
vault 0.9.1

### PR DESCRIPTION
**Pull request by commitay**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by commitay** [comment]
Updated to depend on `gox` https://github.com/Homebrew/homebrew-core/pull/22005

File: Formula/vault.rb [hunk 1]
```diff
@@ -40,12 +29,6 @@ def install
 
     ENV.prepend_create_path "PATH", buildpath/"bin"
```
Comment (ilovezfs) [inline Formula/vault.rb]: Is this still needed?

Comment (commitay) [reply Formula/vault.rb]: Build errors if it is removed.
```
-->    darwin/amd64: github.com/hashicorp/vault
cp: directory /private/tmp/vault-20171222-45675-j9yhoh/bin does not exist
make: *** [dev] Error 1
```

Comment (ilovezfs) [reply Formula/vault.rb]: try just `(buildpath/"bin").mkpath` and then maybe this gets reported lol

Comment (commitay) [reply Formula/vault.rb]: > `(buildpath/"bin").mkpath`

Works.

Comment (ilovezfs) [reply Formula/vault.rb]: #sad

**Review by ilovezfs — APPROVED** [review]

**Comment by JCount** [comment]
Thank you, @commitay!! 😄

**Pull request closed without merging**